docs/modules/ldap.rst

```diff
@@ -0,0 +1,44 @@
+.. _ldap:
+
+====
+LDAP
+====
+
+The LDAP Module provides the ability to authenticate users from an LDAP Identity provider.
+
+Configuration
+-------------
+
+  ldap.provider.base
```

Comment:
for these, gotta look back at the code and see if we assigned defaults to optional ones, otherwise spring injection will throw errors if they're not supplied